PPM Never Have I Ever.py

```diff
@@ -10,19 +10,27 @@
 
 @app.route("/question")
 def question():# df):
+    choose = random.choice(Qs)
+    Qs.remove(choose)
     return '<br><br><br><br><br><br><br><br><br><br><br><center><h1>{}</h1></center>'.format(random.choice(Qs))
 
 @app.route("/player")
 def player():# df):
+    player_name = random.choice(Player)
+    Player.remove(player_name)
     return '<br><br><br><br><br><br><br><br><br><br><br><center><h1>{}</h1></center>'.format(random.choice(Player))
```

Comment:
We can use the variable player_name inside the format method, no need to call random.choice()